test/e2e/serial/config/config.go

```diff
@@ -67,3 +76,62 @@ func GetRteCiImage() string {
 	}
 	return nropTestCIImage
 }
+
+func CheckNodesTopology() error {
+	kconfigs, err := validator.GetKubeletConfigurationsFromTASEnabledNodes(context.TODO(), e2eclient.Client)
```

Comment:
let's create the context from the top level, so this function should accept a context and pass it through